fusebox_connector.py

```diff
@@ -120,8 +124,7 @@ def _build_append_payload(self, playbook_name, unique_string):
         return data
 
     def _handle_check_fuse(self, param):
-        self.debug_print("In action handler for: {0}".format(self.get_action_identifier()))
```

Comment:
@zamastyle, I added this debug_print() function only for print logs, and also we follow the standard that we need to use a minimum of two log statements[either debug_print() or save_progress()] into one function.
It dumps a pretty printed version of the 'dump_object' in the <syslog>/phantom/spawn.log file, where <syslog> typically is /var/log/.
It will not harm anything with your code.

Reply:
As per discussion with @mattsayar-splunk, we keep min_phantom_version as 4.9.0 and print statements as the @zamastyle want to use.